tests/api.c

```diff
@@ -1352,31 +1352,24 @@ static int test_wolfSSL_CTX_load_system_CA_certs(void)
 
 #if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && !defined(NO_WOLFSSL_CLIENT)
     WOLFSSL_CTX* ctx;
+    byte dirValid = 0;
 
     ctx = wolfSSL_CTX_new(wolfSSLv23_client_method());
     if (ctx == NULL) {
         ret = -1;
     }
     if (ret == 0) {
-#ifdef _WIN32
-        if (wolfSSL_CTX_load_system_CA_certs(ctx) != WOLFSSL_NOT_IMPLEMENTED) {
-            ret = -1;
-        }
-#ifdef OPENSSL_EXTRA
-        if (wolfSSL_CTX_set_default_verify_paths(ctx) != WOLFSSL_FAILURE) {
-            ret = -1;
-        }
-#endif /* OPENSSL_EXTRA */
-#else
+    #ifdef _WIN32
```

Comment:
You might consider using `USE_WINDOWS_API`. Some targets report `_WIN32`, but don't really support "windows".

Reply:
Sure. I also realized we don't actually build api.c for Windows, as far as I can tell from looking at our VS solutions.